libs/sql-parser/src/test/java/io/crate/sql/parser/TestSqlParser.java

```diff
@@ -459,16 +459,18 @@ private void assertInstanceOf(String expr, Class<? extends Node> cls) {
 
     @Test
     public void testStackOverflowExpression() {
+        final int copies = isAppleSilicon() ? 6000 : 4000;
```

Comment:
I could avoid those and simply increase the "copies"

Reply:
I'd tend towards increasing it. The failure will short-circuit anyways so shouldn't be slower.

Reply:
Overhead could be from creating a larger string, therefore the conditional copies, but I agree to remove and just increase.